docs/photoniq/event-delivery/getting-started-event-delivery.md

```diff
@@ -0,0 +1,196 @@
+  
+- **Event:** Each new data point or message generated by the publisher.
+
+
+## Setup event stream in the PhotonIQ GDN
+
+**Prerequisites**
+- A Macrometa account with sufficient permissions to create [collections](../../collections/). You can contact a Macrometa partner to sign up or [sign up](https://www.macrometa.com/sign-up) on your own. Once your Macrometa account is set up, you can choose the type of collection to stream your data into. The PhotonIQ GDN offers the following options for EDS:
+    - [Document Collection](../../collections/documents/index.md): Accepts any document type.
+    - [Key-Value Collection](../../collections/keyvalue/index.md): Accepts key-value pairs and can be configured to include blobs.
+
+:::tip
+
+Consult your Macrometa partner to help decide the best type for your business use case.
+
+:::
+
+Now, proceed to set up your collection stream in the GDN following these steps:
+
+1. [Create a Fabric](../../geofabrics/create-geofabric.md) for your collection to specify where your data resides. If you skip this step, the collection will be placed in the system fabric.
+2. [Create a Collection](../../collections/index.md) to receive streaming data. For this guide, we'll be using [Document Collection](../../collections/documents/index.md).
+
+![New Collection Stream](/img/photoniq/event-delivery/new-collection-stream.png)
+
+
+:::important
+
+Ensure you enable **Collection streams** and select a **Distribution** when creating the collection.
+
+:::
+
+
+3. Now that you've succesfully created a collection, add some data to that collection. Adding data to the collection depends on the type of [collection](../../collections/index.md) you created. 
+
+- Navigate to the collection you created and click **New Document**
+- Enter a key or leave empty for an auto-generated key.
+- To add data to this document, toggle the **Tree** option to **Code** and add the following JSON:
+
+```json
+{
+  "first_name": "Jenny",
+  "service": "Executive",
+  "attendance":4
+}
+```
+- Click **Save**
+- You can create multiple documents if you choose. Each of them will have a unique key.
+
+You can also navigate to **Settings** to view more details about the collection.
+
+## Subscribe to events and recieve real time updates
+
+Once you've [set up your event collection stream](#setup-event-stream-in-the-photoniq-gdn), you can subscribe to the Event Delivery Service (EDS) to receive live updates using the [Event Delivery API](https://www.macrometa.com/docs/apiEds#/). 
+
+### Subscribing to the EDS
+
+[Subscribe to the stream](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get) by sending an API request. To subscribe to the stream, it requires the following fields:
+
+- **EDS host**: The host where the EDS service is running.
+- **x-customer-id**: The is used to authenticate the request.
```

Comment:
We should be using validate rather than authenticate as for that we have a API key